src/lib/prospectus.ts

```
@@ -0,0 +1,219 @@
+    return data as ProspectusMetrics;
+}
+
+/**
+ * Generate a base64-encoded SVG bar chart for member growth.
+ * This avoids the need for server-side canvas rendering.
+ */
+export function generateGrowthChartSVG(
+    data: { year: number; members: number }[],
+    primaryColor: string = "#6366f1"
+): string {
+    if (!data || data.length === 0) {
```

ESLint

[error] Delete `··`

(prettier/prettier)